### DISCUSSION TITLE
environmental telemetry if Power Saving is enabled in CLIENT/ROUTER role

### DISCUSSION THREAD

**Discussion by h0lad** [meshtastic/firmware#5117] [Ideas]
### Category

Other

### Hardware

Other

### Firmware Version

2.5.x

### Description

Tested with: Heltec HT-CT62

If Power Saving is enable Telemetry reporting  become  unreliable. Often you only will 1 or 2 telemetry packets being send to the mesh and then it stops.

**How to reproduce**

1. Set CLIENT or ROUTER role
2. sds_secs=ONE_DAY, ls_secs=ONE_DAY,min_wake_secs=1
3. Reduce Telemetry broadcast interval to a "testable" number like 1800
4. Also activate Environmental Telemetry with a sensor

**Result**

You will see 1 or rarely 2 broadcasts. Then nothing.

Client/Router functionality works without issues.

 
Cross-check with disabling "Power Saving" fixes the issues.

**Comment by thebentern** [comment]
The device does not wake up to send telemetry. If you set power savings mode on, the device is likely just asleep for those interval broadcasts

**Reply by h0lad** [reply]
Hm.. could we introduce an additional, Light-Sleep surviving check if telemetry is due?

So if the Router or Client  gets a LoRa interrupt - additionally send telemetry?

**Comment by Dude5101** [comment]
+1 on this

Telemetry should also wake from light sleep

**Reply by h0lad** [reply]
https://github.com/meshtastic/firmware/issues/5134 Feature Ticket there.
